core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -32,11 +32,26 @@ oppia.factory('UrlInterpolationService', [
       }
     };
 
-    var getCachePrefixedUrl = function(resourcePath) {
+    var getPrefixedUrl = function(resourcePath) {
       validateResourcePath(resourcePath);
       return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
     };
 
+    var getUrlWithSlug = function(resourcePath) {
+      validateResourcePath(resourcePath);
+      stripedPath = resourcePath.substring(1);
+      if (GLOBALS.MINIFICATION || !GLOBALS.DEV_MODE) {
+        if (!angular.isUndefined(hashes[stripedPath])) {
+          hash = hashes[stripedPath];
+          splitedPath = resourcePath.split('.');
+          splitedPath.splice(splitedPath.length - 1, 0, hash);
+          resourcePath = splitedPath.join('.');
+          return resourcePath;
+        }
+      }
+      return resourcePath;
```

Comment:
The 0 was excluded only for hashes indexing, the returned string contained the 0 character.